radiant/gtkdlgs.cpp

```diff
@@ -374,6 +374,12 @@ game_t gameList[] = {
 	{ "sof2.game", sSOF2ComboItem, "base", qtrue, qfalse },
 	{ "sof2.game", sSOF2ModComboItem, "", qfalse, qtrue },
 
+	{ "q2.game", "Quake II", "baseq2", qtrue, qfalse },
+	{ "q2.game", "Capture The Flag", "ctf", qfalse, qfalse },
+	{ "q2.game", "Quake II Mission Pack: The Reckoning", "xatrix", qfalse, qfalse },
+	{ "q2.game", "Quake II Mission Pack: Ground Zero", "rogue", qfalse, qfalse },
```

Comment:
Total nitpick, but you might as well put Rogue before Xatrix and then they're all alphabetical, except for Custom, which should be last ;)